docs/ref/request-response.txt

```diff
@@ -735,16 +736,15 @@ Attributes
 Methods
 -------
 
-.. method:: HttpResponse.__init__(content='', content_type=None, status=200, reason=None, charset=None)
+.. method:: HttpResponse.__init__(content=b'', content_type=None, status=200, reason=None, charset=None)
 
     Instantiates an ``HttpResponse`` object with the given page content and
     content type.
 
-    ``content`` should be an iterator or a string. If it's an
-    iterator, it should return strings, and those strings will be
-    joined together to form the content of the response. If it is not
-    an iterator or a string, it will be converted to a string when
-    accessed.
+    ``content`` should be an iterator, bytestring, or string. If it's an
+    iterator, it should return strings or bytestrings, and those will be
+    joined together to form the content of the response. Other types will be
```

Comment:
I'm still not sure what "Other types" is referring to. Non-iterators? Or is it saying that you can pass types besides "iterator, bytestring, or string"? (Now reading the old version, I see the latter...)

Perhaps this would be more clear:
``content`` is most commomly an iterator, bytestring, or string. Other types
will be converted to a bytestring by encoding their string representation.
Iterators should return strings or bytestrings and those will be joined
together to form the content of the response.

Reply:
It sounds better after your rephrasing (like always :1st_place_medal: ).